nuxeo-runtime/nuxeo-runtime-stream/src/main/resources/OSGI-INF/avro-service.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component name="org.nuxeo.runtime.avro" version="1.0">
+
+  <documentation>
+    The Avro component.
+    It provides services 
+    - that allow to register and retreives schemas
+    - that allow to create avro schemas from custom types
+    - that allow to de/serialize avro records from/to custom types 
+  </documentation>
+
+  <implementation class="org.nuxeo.runtime.avro.AvroComponent" />
+
+  <service>
+    <provide interface="org.apache.avro.message.SchemaStore" />
+    <provide interface="org.nuxeo.runtime.avro.AvroSchemaStoreService" />
+    <provide interface="org.nuxeo.runtime.avro.AvroFactoryService" />
+  </service>
+
+  <extension-point name="schema">
+    <object class="org.nuxeo.runtime.avro.AvroSchemaDescriptor" />
```

Comment:
Your `extension-point` should contain a `documentation` tag describing how to contribute to it. You can find an example in `versioning-service.xml`.

They will be exported latter to explorer.